frontend/src/components/GameView/GameView.js

```diff
@@ -18,12 +20,17 @@ export default function GameView() {
     [1, 2, 3, 4, 5, 6, 7, 8, 9]
   ];
 
+  const mockTime = { max_time: 300, current_time: 6 };
```

Comment:
This is not enforced, but try to keep one convention of naming. In this case I would suggest changing the snake case to camel case.

Reply:
Also, doesn't make sense to put those values in an object if you are gonna pass them separately anyway. Either make the prop accept the whole object or put them in two separate values.

Reply:
Or another solution is to name the object keys accordingly to props so: `const mockTime = { currentTime: 6, maxTime: 300 }` and then you can pass it as props like this: `<Timer {...mockTime} />`

Reply:
Changed as suggested.